src/Generator/BooleanGenerator.php

```diff
@@ -11,10 +10,10 @@ function bool()
 
 class BooleanGenerator implements Generator
 {
-    public function __invoke($_size, $rand)
+    public function __invoke($_size, \Eris\Random\RandomRange $rand)
```

Comment:
for consistency this should be an additional `use` statement at the top of the file